server/core/store.go

```diff
@@ -103,15 +104,50 @@ func (s *StoreInfo) DownTime() time.Duration {
 }
 
 const minWeight = 1e-6
+const maxScore = 1024 * 1024 * 1024
 
-// LeaderScore returns the store's leader score: leaderCount / leaderWeight.
-func (s *StoreInfo) LeaderScore() float64 {
-	return float64(s.LeaderSize) / math.Max(s.LeaderWeight, minWeight)
+// LeaderScore returns the store's leader score: leaderSize / leaderWeight.
+func (s *StoreInfo) LeaderScore(delta int64) float64 {
+	return float64(s.LeaderSize+delta) / math.Max(s.LeaderWeight, minWeight)
 }
 
-// RegionScore returns the store's region score: regionSize / regionWeight.
-func (s *StoreInfo) RegionScore() float64 {
-	return float64(s.RegionSize) / math.Max(s.RegionWeight, minWeight)
+// RegionScore returns the store's region score.
+func (s *StoreInfo) RegionScore(highSpaceRatio, lowSpaceRatio float64, delta int64) float64 {
+	if s.RegionSize == 0 {
+		return float64(delta)
+	}
+
+	capacity := float64(s.Stats.GetCapacity()) / (1 << 20)
+	available := float64(s.Stats.GetAvailable()) / (1 << 20)
+
+	var score float64
+
+	// lowSpaceRatio shouldn't greater than highSpaceRatio
+	if lowSpaceRatio > highSpaceRatio {
```

Comment:
we can check this in config adjustment.

Reply:
using pd-ctl to set config will not call adjust()

Reply:
`lowSpaceRatio = highSpaceRatio` will panic in the formula

Reply:
when `lowSpaceRatio == highSpaceRatio`, it is impossible to run into else branch.